plugins/text-editor-resources/src/provider/utils.ts

```diff
@@ -36,28 +36,30 @@ export function createLocalProvider (ydoc: Ydoc, doc: CollaborativeDoc): Provide
 }
 
 export function createRemoteProvider (ydoc: Ydoc, doc: CollaborativeDoc, content: Ref<Blob> | null): Provider {
-  const collaborator = getMetadata(textEditor.metadata.Collaborator)
-
   const token = getMetadata(presentation.metadata.Token) ?? ''
   const collaboratorUrl = getMetadata(presentation.metadata.CollaboratorUrl) ?? ''
 
   const documentId = getDocumentId(doc)
 
-  return collaborator === 'cloud'
-    ? new CloudCollabProvider({
-      url: collaboratorUrl,
-      name: documentId,
-      document: ydoc,
-      content,
-      token
-    })
-    : new HocuspocusCollabProvider({
-      url: collaboratorUrl,
-      name: documentId,
-      document: ydoc,
-      token,
-      parameters: { content }
-    })
+  const provider = new HocuspocusCollabProvider({
+    url: collaboratorUrl,
+    name: documentId,
+    document: ydoc,
+    token,
+    parameters: { content },
+    onConnect: () => {
+      void setPlatformStatus(OK)
```

Comment:
May be better to show a placeholder inside editor? Any other service could clear status